Chef.PowerShell.Wrapper/Chef.PowerShell.Wrapper.cpp

```diff
@@ -35,11 +34,26 @@ Assembly^ currentDomain_AssemblyResolve(Object^ sender, ResolveEventArgs^ args)
     return nullptr;
 }
 
-const wchar_t* ExecuteScript(const char* powershellScript, int timeout)
+// This is the entry point for the DLL. It is called from ruby with the powershell script to execute.
+// Note that this is for "PowerShell" (5.1 and earlier) and not "PowerShell Core" (6.0 and later).
+// You likely want to make similiar changes to the Chef.PowerShell.Core.Wrapper.cpp file.
```

Comment:
I'd remove this comment line.

Reply:
Same comment as before... warning for future editors of this gem.